src/mpl/include/mpl_sysconf.h

```diff
@@ -0,0 +1,16 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/* vim: set ft=c.mpich : */
+/*
+ *  (C) 2018 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#ifndef MPL_SYSCONF_H_INCLUDED
+#define MPL_SYSCONF_H_INCLUDED
+
+#include "mplconfig.h"
+
+/* Returns the number of processors currently available in the system */
+int MPL_get_nprocs(void);
+
+#endif /* MPL_SYSCONF_H_INCLUDED */
```

Comment:
How about `mpl_misc.h` and `src/misc/mpl_misc.c`? `sysconf` seems cryptic.

Reply:
moved to `mpl_misc`